src/frontends/tensorflow_common/src/op/matrix_inverse.cpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/op/inverse.hpp"
+
+#include "common_op_table.hpp"
+
+using namespace std;
+using namespace ov::op;
+
+namespace ov {
+namespace frontend {
+namespace tensorflow {
+namespace op {
+OutputVector translate_matrix_inverse_op(const NodeContext& node) {
+    // a function that performs standard or default checks on a specific operation within the OpenVINO pipeline
+    default_op_checks(node, 1, {"GELU"}); // copied from issue #22964
+    auto op_type = node.get_op_type();
+    TENSORFLOW_OP_VALIDATION(node, op_type == "MatrixInverse", "Internal error: incorrect usage of translate_matrix_inverse_op.");
+    auto input = node.get_attribute<std::string>("input"); // extract the input attribute from tf.raw_ops.MatrixInverse(input, adjoint=False, name=None)
```

Comment:
only "adjoint" attribute needs to be handled. input is an input tensor

Reply:
OK, I see. : )